changes/7860.bugfix

```diff
@@ -0,0 +1 @@
+Improper CLI Command for creating a new migration script
```

Comment:
We dont need the bugfix as it is only documenation update and can make confusion that the actual command was wrong. 

Dont worry if there is error complaingin about missing bugfix file

Reply:
@tino097 , I have delete the bugfix file. Please merge this PR if it looks good. Thanks :)